contracts/0.7.x/libraries/Binary/BinaryArbitrable.sol

```diff
@@ -124,7 +117,7 @@ library BinaryArbitrable {
     ) internal returns(uint256 disputeID) {
 
         ItemData storage item = self.items[_itemID];
```

Comment:
This method's body feels a bit messy. How about rearranging lines and adding some blank ones?

```solidity
ItemData storage item = self.items[_itemID];
require(item.status == Status.None, "Item already disputed.");

disputeID = self.arbitrator.createDispute{value: _arbitrationCost}(AMOUNT_OF_CHOICES, self.arbitratorExtraData);

item.status = Status.Disputed;
item.disputeID = disputeID;
item.rounds.push();

self.disputeIDtoItemID[disputeID] = _itemID;

emit Dispute(self.arbitrator, disputeID, _metaEvidenceID, _evidenceGroupID);
```

Reply:
Done